stan/math/rev/core/operator_unary_negative.hpp

```diff
@@ -44,10 +45,12 @@ class neg_vari : public op_v_vari {
    \end{cases}
    \f]
  *
+ * @tparam A var autodiff type
  * @param a Argument variable.
  * @return Negation of variable.
  */
-inline var operator-(const var& a) {
+template <typename Var, require_var_t<Var>...>
+inline var operator-(Var&& a) {
   return var(new internal::neg_vari(a.vi_));
```

Comment:
[optional as it wasn't in your code]
Just wanted to throw this out for the future.  When we don't return refs, we can just use the braces constructor:

```
return { new internal::neg_vari(a.vi_) };
```

Reply:
Will do!